schema.graphql

```diff
@@ -552,12 +552,43 @@ type Transfer @entity(immutable: true) {
   from: Bytes!
 }
 
+type Dependency @entity {
+  id: ID!
```

Comment:
nit: can we add a comment specifying the id syntax (for this and the other new ones)?

Reply:
Resolved in https://github.com/ArtBlocks/artblocks-subgraph/pull/168